### PR TITLE
seperating multiple email addresses in x509->email with a space

### PR DESCRIPTION
**Pull request by errror**
seperating multiple email addresses in x509->email with a space instead of just concatenating them without seperator

# Description

x509->email concatenates multiple email addresses without a seperator. This patch introduces space (' ') as seperator in order for consumers to extract the multiple email address with `split(/\s/)`.

Fixes/addresses (If applicable) # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Changes have been manually tested (please provide information on test platform using the fields below)

## Test / Development Platform Information

- Debian Bullseye
- Crypt::OpenSSL::X509 commit 042019d8ee5d827b1993707d6694bb71e54fe97c (tag: 1.9.8)
- Perl version v5.32.1
- OpenSSL version 1.1.1k  25 Mar 2021

### CODE REVIEW

File: certs/smime.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+openssl req -x509 -nodes -days 730 -newkey rsa:2048 -keyout /dev/null -out smime.pem -config smime.conf -extensions 'v3_req'
```
Comment (jonasbn) [inline certs/smime.sh:3]: Could you offer a comment on what this does?

**Review by jonasbn — CHANGES_REQUESTED** [review]
Hi @errror 

Thanks for the contribution, please see my request for a change

File: certs/smime.sh (hunk 1 above)
Comment (errror) [reply certs/smime.sh:3]: > Could you offer a comment on what this does?

Sure. It is a small script how to generate the smime.pem which I included in the patch. smime.pem is used in my added unit tests.

Comment (jonasbn) [reply certs/smime.sh:3]: Thanks

**Review by jonasbn — APPROVED** [review]
Approved

**Pull request merged**